docs/src/writing-tests-java.md

```diff
@@ -0,0 +1,94 @@
+---
+id: writing-tests
+title: "Writing Tests"
+---
+
+Playwright assertions are created specifically for the dynamic web. Checks are automatically retried until the necessary conditions are met. Playwright comes with [auto-wait](./actionability.md) built in meaning it waits for elements to be actionable prior to performing actions. Playwright provides [assertThat](./test-assertions.md) overloads to write assertions.
```

Comment:
this needs to be rephrased to provide some intro, perhaps "Playwright comes with built in assertions which are created specifically...." Otherwise it sounds as if an intro section was somewhere and this is just a continuation.

Also it's unclear if auto-wait applies just to the assertions or to the library or both. We can try to rewrite this paragraph later if you want.

Reply:
we have it in all the intros, lets follow up after it